modules/sdk/project-templates/project-templates-form-field/build.gradle

```diff
@@ -2,12 +2,22 @@ dependencies {
 	compileOnly group: "com.beust", name: "jcommander", version: "1.72"
 	compileOnly group: "com.liferay", name: "com.liferay.project.templates.extensions", version: "1.0.17"
 	compileOnly group: "org.apache.maven.archetype", name: "archetype-common", version: "2.4"
+	compileOnly project(":core:petra:petra-string")
+
+	testCompile group: "biz.aQute.bnd", name: "biz.aQute.bnd", version: "3.5.0"
+	testCompile group: "com.googlecode.java-diff-utils", name: "diffutils", version: "1.3.0"
+	testCompile group: "com.liferay", name: "com.liferay.maven.executor", version: "1.0.0"
+	testCompile group: "junit", name: "junit", version: "4.12"
+	testCompile group: "net.diibadaaba.zipdiff", name: "zipdiff", version: "1.0"
+	testCompile project(path: ":sdk:project-templates:project-templates", configuration: "testAPI")
+
+	testRuntime project(":sdk:project-templates:project-templates").files("classes")
```

Comment:
@gabrielibson, was it necessary due to the update of task `generateSampleProject` or due to the unit tests of the form-field template?

Reply:
@natocesarrego, this dependency: `compileOnly project(":core:petra:petra-string")`, is related to StringPool.BLANK inside of `FormFieldProjectTemplateCustomizer`.

The other ones are related to unit tests of the form-field template. 
I followed [this](https://github.com/natocesarrego/liferay-portal/blob/master/modules/sdk/project-templates/project-templates-service/build.gradle) example suggested by Lawrence.

Thank you for reviewing.

Reply:
@natocesarrego, it's fixed now. Please, let me know if you find any other inconsistency.

Thanks.